includes/IPBAuth.php

```diff
@@ -233,6 +233,7 @@ public static function updateUser(&$user)
      */
     public static function userExists($username)
     {
+	$cfg = IPBAuth::getConfig();
```

Comment:
Please use spaces instead of a tab (to keep code consistent) otherwise this looks good!